dnf/base.py

```diff
@@ -2169,7 +2169,11 @@ def provides(self, provides_spec):
         providers = dnf.query._by_provides(self.sack, provides_spec)
         if providers:
             return providers, [provides_spec]
-        binary_provides = [prefix + provides_spec for prefix in ['/usr/bin/', '/usr/sbin/']]
+        if provides_spec.startswith('/bin/') or provides_spec.startswith('/sbin/'):
+            binary_provides = ['/usr' + provides_spec]
+        else:
```

Comment:
There could be also a performance improvement using something like:
```
elif provides_spec.startswith('/'):
    return providers, [provides_spec]
```
What do you think @m-blaha ?